workers/ohsome_quality_analyst/base/indicator.py

```diff
@@ -41,19 +38,28 @@ class Result:
         timestamp_oqt (datetime): Timestamp of the creation of the indicator
         timestamp_osm (datetime): Timestamp of the used OSM data
             (e.g. Latest timestamp of the ohsome API results)
-        label (str): Traffic lights like quality label
-        value (float): The result value as float ([0, 1])
-        description (str): Description of the result
+        label (str): Traffic lights like quality label: Green, yellow or red. The value
+            is determined by the result classes
+        value (float): The result value
+        class_ (int): The result class. An integer between 1 and 5. It maps to the
```

Comment:
I'm not really a fan of using the `…_` notation for parameters/variables that are used outside the class. Is there a better option? `result_class` maybe? or `label_class`?